src/fraction/unicode_fromto_str.rs

```diff
@@ -0,0 +1,589 @@
+use std::fmt::Display;
+
+use crate::fraction::GenericFraction;
+use crate::{error::ParseError, Sign};
+use num::rational::Ratio;
+use num::Zero;
+use std::{fmt, str};
+use Integer;
+
+impl<T: Clone + Integer + Display + From<u8>> GenericFraction<T> {
+    /// Display the fraction using FRACTION SLASH '\u{2044}' '⁄'
+    /// If you have font support, this is the way Unicode wants to display fractions
+    /// ```
+    /// use fraction::Fraction;
+    /// assert_eq!(
+    ///   format!("{}", Fraction::new(1u8,2u8).display_unicode()),
+    ///   "1⁄2"
+    /// );
+    /// assert_eq!(
+    ///   format!("{}", Fraction::new(3u8,2u8).display_unicode()),
+    ///   "3⁄2"
+    /// );
+    /// ```
+    pub fn display_unicode(&self) -> impl fmt::Display + '_ {
```

Comment:
Could we rename to `get_unicode_display`? Since the method itself does not produce any side-effects, but rather returns a UnicodeDisplay representation/wrapper.